dev/linearhooks/internal/handlers/handlers.go

```diff
@@ -107,15 +108,37 @@ func (h *Handler) HandleIssueMover(w http.ResponseWriter, r *http.Request) {
 func moveIssue(logger log.Logger, client graphql.Client, rules []RuleSpec, issue linearschema.IssueData) error {
 	for _, r := range rules {
 		if newTeamId := r.moveToNewTeam(issue); newTeamId != nil {
-			logger.Info("moving issue",
-				log.String("from.id", issue.Team.Key),
-				log.String("to", r.Dst.TeamID),
+			logger := logger.With(
+				log.String("issue.id", issue.Identifier),
+				log.String("from.key", issue.Team.Key),
+				log.String("to.id", r.Dst.TeamID),
 			)
+			logger.Info("moving issue")
+
 			teamUUID, err := getTeamUUID(context.Background(), client, r.Dst.TeamID)
 			if err != nil {
 				return errors.Wrapf(err, "get dst team %q UUID", r.Dst.TeamID)
 			}
-			if err := moveIssueToTeam(context.Background(), client, issue.Identifier, teamUUID); err != nil {
+
+			input := moveIssueToTeamInput{
+				issueId: issue.Identifier,
+				teamId:  teamUUID,
```

Comment:
```suggestion
				teamUUID:  teamUUID,
```

It's a bit confusing to use 'id' and 'UUID' interchangeably.